src/main/kotlin/io/github/mojira/arisa/infrastructure/Jira.kt

```diff
@@ -46,6 +46,17 @@ fun addComment(issue: Issue, comment: String) = runBlocking {
     }
 }
 
+// The used library doesn't include visibility information in comments for whatever reason
+fun isCommentRestrictedTo(jiraClient: JiraClient, comment: Comment, group: String) = runBlocking {
+    Either.catch {
+        val commentJson = jiraClient.restClient.get(URI(comment.self)) as JSONObject
+        if (commentJson.contains("visibility")) {
+            val visibility = commentJson.getJSONObject("visibility")
+            visibility.get("type") == "group" && visibility.get("value") == group
+        } else false
+    }
+}
```

Comment:
I really don't like the jira library used here.
I already didn't like having to take care of the json for updating comments, and now it turns out the library classes don't actually include all the information returned by the API.

Reply:
https://github.com/rcarz/jira-client/blob/master/src/main/java/net/rcarz/jiraclient/Comment.java#L41

It does allow to get visibility from the comment, you might need to get the comment by I'd again or add it to the expand so jira returns the info

You can also do .update() to change the visibility

Reply:
The link you posted seems to be a newer version of the library. I already looked at the comment class, it does not contain that method.